### PR TITLE
To do list component

### PR DESCRIPTION
**Pull request by nikitha2000**
list component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,38 +2,49 @@ import React, { useState } from "react"
...
         <div className='app-container'>
           <h1>TODO App</h1>
+
             <div className='layout-buttons-container'>
                <Button color="#8888b9" label="Todo" onClick={() => {layoutToggleButtonClick("todoButton")}}  isClicked={isTodoButtonClicked}  />
                <Button color="#8888b9" label="Todo list" onClick={() => {layoutToggleButtonClick("todoListButton")}}  isClicked={isTodoListButtonClicked} />
             </div>
-
-           {isTodoButtonClicked && (
+
```
Comment (vonnue-nidheesh) [inline src/App.js]: Revert this change

File: src/App.js [hunk 2]
```diff
@@ -2,38 +2,49 @@ import React, { useState } from "react"
...
+              <ToDoList todos={todos} onRemoveTodo={removeTodo} />
+            )}
         </div>
       </div>
     );
-            
 };
 
-export default App;
+export default App;
```
Comment (vonnue-nidheesh) [inline src/App.js]: No need of this space here

File: src/component/Button.jsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from "react"
 
-const Button = ({ label, onClick, color='#8888b9', isClicked=false, isDisabled=false}) => {
+const Button = ({  label, onClick,color='#8888b9', isClicked=false, isDisabled=false}) => {
```
Comment (vonnue-nidheesh) [inline src/component/Button.jsx]: format file

File: src/component/TodoForm.jsx [hunk 4]
```diff
@@ -40,9 +40,9 @@
...
               </div>
-            <Button color='#8888b9' label='Create To Do' onClick={handleSubmit} isDisabled={!taskName} />
+            <Button color='#8888b9' label='Create To Do' onClick={handleSubmit}  isDisabled={!taskName} />
         </form>
         );
     };
-
-export default ToDoForm; 
+                                                                                          
+export default ToDoForm ;
```
Comment (vonnue-nidheesh) [inline src/component/TodoForm.jsx]: Format files

File: src/App.js (hunk 1 above)
Comment (nikitha2000) [reply src/App.js]: i did not understand

Comment (vonnue-nidheesh) [reply src/App.js]: No need for empty space here.

compare with line no 32

**Review by vonnue-nidheesh — APPROVED** [review]
